perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java

```diff
@@ -106,6 +107,21 @@
 	List<Facility> getFacilitiesByDestination(PerunSession perunSession, String destination) throws InternalErrorException, FacilityNotExistsException, PrivilegeException;
 
 	/**
+	 * Returns all facilities who have set the attribute with the value. Searching by attributeName. Searching only def and opt attributes.
+	 * Can find only attributes with String Value by this way! (not Integer, Map or List)
```

Comment:
This is not true, it can handle string, integer, boolean and even list attributes, but the list must be in exactly same order of values as stored value. In a fact, it can't handle large attributes (map, large string, large array/list), because select compares only ``attr_value`` column. Please state, that large attributes are not supported in javadoc through all tree layers. Also please throw exception in Bl and Impl, if passed attribute is of large type.